source/content/guides/wordpress-composer/04-bedrock-composer.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Using Bedrock to Create Your WordPress Pantheon Site
+subtitle: Create a Composer-managed WordPress Site with Bedrock
+description: Learn more about Bedrock for Composer-managed WordPress sites.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/bedrock-composer
+anchorid: bedrock-composer
+---
+
+[Composer](https://getcomposer.org/) does not fully support WordPress, however, Bedrock is a central feature of Composer. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
+
+## Create a New Pantheon WordPress Site with Bedrock
+
+1. Run the following command in Terminus:
+
+    ```bash
+    composer create-project your_site
+    ```
+
+1. Update the [environment variables](https://docs.roots.io/bedrock/master/environment-variables/#wp-env) in your project's `.env` file.
+
+1. Store your `.env` outside of the project root on the Pantheon server. 
```

Comment:
this step seems out of order. All the other steps relating to `.env` are below the step to commit `vendor` and `web`. Also, for the record, I didn't need to _create_ a `.env` file, I just had one, presumably as part of the Bedrock initialization step.